packages/app-headless-cms/src/admin/config/contentEntries/list/Browser/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Property, useIdGenerator } from "@webiny/react-properties";
+import { useModel } from "~/admin/hooks";
+
+export interface FilterConfig {
+    name: string;
+    element: React.ReactElement;
+}
+
+export interface FilterProps {
+    name: string;
+    element?: React.ReactElement<unknown>;
+    modelIds?: string[];
+    remove?: boolean;
+    before?: string;
+    after?: string;
+}
+
+export const Filter: React.FC<FilterProps> = ({
```

Comment:
Feel free to use the `React.VFC` instead of `React.FC`.

Reply:
We'll make this component support decorators, which will force this to become a composable component, and in that case, we can't use FC/VFC.